doc/releases/migration-guide-3.7.rst

```diff
@@ -451,6 +451,24 @@ Bluetooth Audio
   the :c:func:`bt_bap_stream_connect` shall now be called before :c:func:`bt_bap_stream_start`.
   (:github:`73032`)
 
+* Renamed ``stream_lang`` to just ``lang`` to better fit with the assigned numbers document.
+  This affects the ``BT_AUDIO_METADATA_TYPE_LANG`` macro and the following functions:
+  - :c:func:`bt_audio_codec_cap_meta_set_lang`
```

Comment:
If this was meant to appear as a list, it does not :) 
https://builds.zephyrproject.io/zephyr/pr/72584/docs/releases/migration-guide-3.7.html#bluetooth-audio
Is the () in the same indent rendering properly?

Reply:
Yes the intention was to make it a list - I guess I went with a markdown list instead of a RST list :D 

Thanks